client/components/PageList/PageList.jsx

```diff
@@ -58,7 +15,63 @@ function linkIf(condition, props, children) {
   return children;
 }
 
-class Pager extends React.Component {
+const PageLink = (props) => (
+  <ListItem key={props.pageNum}>
+    <Link
+      onClick={(e) => {
+        e.preventDefault();
+        props.handleClick(props.pageNum);
+      }}
+      className={props.activePage === props.pageNum && 'active'}
+      href="."
+    >
+      {props.pageNum}
+    </Link>
+  </ListItem>
+);
+
+class PageList extends React.Component {
+  static collapsePages(pages, activePage, handleClick) {
+    if (pages.length <= 3) {
+      return pages.map((pageNum) => (
+        <PageLink
+          key={pageNum}
+          pageNum={pageNum}
+          activePage={activePage}
+          handleClick={handleClick}
+        />
+      ));
+    }
+    if (activePage <= 2) {
+      return (
+        <>
+          <PageLink pageNum={1} activePage={activePage} handleClick={handleClick} />
+          <PageLink pageNum={2} activePage={activePage} handleClick={handleClick} />
+          <Ellipsis>…</Ellipsis>
+          <PageLink pageNum={pages.length} activePage={activePage} handleClick={handleClick} />
+        </>
```

Comment:
Sure! They're called "fragments". You know how React makes you wrap you components in render in one container element? If you don't want that an extra element to be around your components you can use this syntax instead of using a div for example.
<> </> will not render anything, only your components inside. https://reactjs.org/docs/fragments.html#short-syntax